app/controllers/users_controller.rb

```diff
@@ -21,6 +24,23 @@ def new
     @user = User.new
   end
 
+  def edit
+    @user = User.find(params[:id])
+
+    @stories = @user.stories.order(created_at: :desc)
+
+    stories_json = @stories.map do |story|
+      {
+        user_id: @user.id,
+        id: story.id,
+        name: story.body,
+        day: story.day,
+        images: story.images.attached? ? rails_blob_path(story.images.first, only_path: true) : nil
+      }
+    end
+
+    render json: { stories: stories_json }
+  end
   def edit; end
```

Comment:
Why do you have the same method but empty?

Reply:
For me I dont see any method empty but I will check again. Thanks